File: spring-github-oauth2/src/main/java/com/knf/dev/demo/springoauth2github/config/WebMvcConfig.java
package com.knf.dev.demo.springoauth2github.config;

import org.springframework.context.annotation.Configuration;
import org.springframework.web.servlet.config.annotation.ViewControllerRegistry;
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;

@Configuration
public class WebMvcConfig implements WebMvcConfigurer {

    @Override
    public void addViewControllers(ViewControllerRegistry registry){
        registry.addViewController("/").setViewName("index");
        registry.addViewController("/index").setViewName("index");
    }
}

File: kotlin-spring-main/kotlinspringOauth2google/src/main/kotlin/com/knf/dev/demo/kotlinspringOauth2google/config/WebMvcConfig.kt
package com.knf.dev.demo.kotlinspringOauth2google.config

import org.springframework.context.annotation.Configuration
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
import org.springframework.web.servlet.config.annotation.ViewControllerRegistry

@Configuration
class WebMvcConfig : WebMvcConfigurer {
    override fun addViewControllers(registry: ViewControllerRegistry) {
        registry.addViewController("/").setViewName("index")
        registry.addViewController("/index").setViewName("index")
    }
}